<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>perturbationForcesDialog</class>
 <widget class="QDialog" name="perturbationForcesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>651</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>651</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Gravity</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QRadioButton" name="GravityFieldRadioButton">
              <property name="text">
               <string>Gravity effects</string>
              </property>
              <property name="icon">
               <iconset resource="../../iconary/sta-icons.qrc">
                <normaloff>:/icons/Gravity_field_icon.png</normaloff>:/icons/Gravity_field_icon.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="GravLabel">
                <property name="text">
                 <string>Model:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="ZonalsLabel">
                <property name="text">
                 <string>Zonals: </string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="ZonalsSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="minimum">
                 <number>2</number>
                </property>
                <property name="maximum">
                 <number>70</number>
                </property>
                <property name="value">
                 <number>2</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="TesseralLabel">
                <property name="text">
                 <string>Tesserals: </string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="GravityEffectsModelComboBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Drag</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="AtmDragRadioButton">
              <property name="text">
               <string>Atmospheric Drag</string>
              </property>
              <property name="icon">
               <iconset resource="../../iconary/sta-icons.qrc">
                <normaloff>:/icons/Mydrag.png</normaloff>:/icons/Mydrag.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QFormLayout" name="formLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="AthmLabel_4">
                  <property name="text">
                   <string>Model type: </string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="AtmosphereDragTypeComboBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="dragDayF10Edit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="dragAvF10Edit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="dragAvF10Label">
                  <property name="text">
                   <string>Average F10.7:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="dragGeomagLabel">
                  <property name="text">
                   <string>Geomagnetic Index:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="dragGeomagEdit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="dragDayF10Label">
                  <property name="text">
                   <string>Daily F10.7:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Other forces</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="0">
             <widget class="QRadioButton" name="SolarPressureRadioButton">
              <property name="text">
               <string>Solar Pressure</string>
              </property>
              <property name="icon">
               <iconset resource="../../iconary/sta-icons.qrc">
                <normaloff>:/icons/Solar_pressure_icon.png</normaloff>:/icons/Solar_pressure_icon.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="DebrisRadioButton">
              <property name="text">
               <string>Micrometeoroids / Debris</string>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QCheckBox" name="AlbedoCheckBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Albedo</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>55</width>
                  <height>55</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="IRCheckBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>IR</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>55</width>
                  <height>55</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="CrLabel">
                  <property name="text">
                   <string>Cr:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="CrSpinBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="decimals">
                   <number>2</number>
                  </property>
                  <property name="maximum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.010000000000000</double>
                  </property>
                  <property name="value">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>3rd bodies</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="ThirdBodyRadioButton">
            <property name="text">
             <string>3rd Body</string>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icons/3rdbody_icon.png</normalon>
             </iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QListWidget" name="BodyListWidget">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <item>
               <property name="text">
                <string>Sun</string>
               </property>
               <property name="font">
                <font>
                 <italic>false</italic>
                </font>
               </property>
               <property name="background">
                <brush brushstyle="NoBrush">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboSun.png</normaloff>:/icons/ComboSun.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Earth</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboEarth.png</normaloff>:/icons/ComboEarth.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Mercury</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboMercury.png</normaloff>:/icons/ComboMercury.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Venus</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboVenus.png</normaloff>:/icons/ComboVenus.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Mars</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboMars.png</normaloff>:/icons/ComboMars.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Jupiter</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboJupiter.png</normaloff>:/icons/ComboJupiter.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Saturn</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboSaturn.png</normaloff>:/icons/ComboSaturn.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Uranus</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboUranus.png</normaloff>:/icons/ComboUranus.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Neptune</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboNeptune.png</normaloff>:/icons/ComboNeptune.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Pluto</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboPluto.png</normaloff>:/icons/ComboPluto.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Moon</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboMoon.png</normaloff>:/icons/ComboMoon.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Deimos</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboDeimos.png</normaloff>:/icons/ComboDeimos.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Europa</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboEuropa.png</normaloff>:/icons/ComboEuropa.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Titan</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboTitan.png</normaloff>:/icons/ComboTitan.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Enceladus</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboEnceladus.png</normaloff>:/icons/ComboEnceladus.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Triton</string>
               </property>
               <property name="icon">
                <iconset resource="../../iconary/sta-icons.qrc">
                 <normaloff>:/icons/ComboTriton.png</normaloff>:/icons/ComboTriton.png</iconset>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="AddPushButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../../iconary/sta-icons.qrc">
                  <normaloff>:/icons/timeline-double.png</normaloff>:/icons/timeline-double.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="RemovePushButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../../iconary/sta-icons.qrc">
                  <normaloff>:/icons/timeline-half.png</normaloff>:/icons/timeline-half.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="PertBodyListWidget">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../iconary/sta-icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>perturbationForcesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>perturbationForcesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
